main.go

```diff
@@ -261,9 +263,11 @@ func setBuildFlags(flags *flag.FlagSet, b *buildCmd) {
 	flags.BoolVar(&b.parseOnly, "parse-only", false, "exit after dumping parse result")
 	flags.BoolVar(&b.codeGenOnly, "codegen-only", false, "codegen only, don't compile")
 	flags.BoolVar(&b.compileOnly, "compile-only", false, "compile only, don't start web server after")
-	flags.StringVar(&b.outDir, "out-dir", "./build", "path to output build directory")
+	flags.StringVar(&b.outDir, "out-dir", "./build", "path to output build directory. Defaults to ./build")
+	flags.StringVar(&b.outFile, "out-file", "", "path to output application binary. Defaults to ./build/bin/projectName.exe")
```

Comment:
The `.exe` thing was just a random thing I did, I never intended for it to stick around. Now's probably a good time to remove it. I don't want to cause any confusion about Windows executables and whatnot.